vendor/noise_handshake/src/handshake.rs

```diff
@@ -99,8 +99,6 @@ impl<IO: AsyncRead + AsyncWrite> Future for NoiseHandshake<IO> {
     type Error = std::io::Error;
 
     fn poll(&mut self) -> Poll<Self::Item, Self::Error> {
-        let msg = format!("{} [{}] Polling", Utc::now(), self.name);
-        dbg!(msg);
         match self {
             Self {
                 name,
```

Comment:
:x: I think the name was only used for debugging.

Reply:
Yep, removed :wastebasket: